<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rt_dt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="createSystemBtn">
      <property name="text">
       <string>System design</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openCalculationEqualReliabilityBtn">
      <property name="text">
       <string>Calculation (A(1) = P(1))</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openCalculationGreaterReliabilityBtn">
      <property name="text">
       <string>Calculation (A(1) &gt;= P(1))</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeBtn">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>182</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
